graphql/patch_resolver.go

```diff
@@ -59,10 +59,14 @@ func (r *patchResolver) CommitQueuePosition(ctx context.Context, obj *restModel.
 	if *obj.Alias == evergreen.CommitQueueAlias {
 		cq, err := commitqueue.FindOneId(*obj.ProjectId)
 		if err != nil {
-			return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error getting commit queue position for patch %s: %s", *obj.Id, err.Error()))
+			return nil, InternalServerError.Send(ctx, fmt.Sprintf("getting commit queue position for patch '%s': %s", *obj.Id, err.Error()))
 		}
 		if cq != nil {
 			position := cq.FindItem(*obj.Id)
+			// Adjust to be one-indexed if found on commit queue.
+			if position >= 0 {
+				position += 1
```

Comment:
Would it make sense to just make this change on the front end instead? 
On the commit queue page we increment the `commitQueuePosition` by 1 before rendering it i think we should be consistent with how we handle/return this value.
https://github.com/evergreen-ci/spruce/blob/71b8a6253f06da15d08562866aae25c5854f9608/src/pages/CommitQueue.tsx#L60

Reply:
I don't necessarily think it would be more consistent because the code linked technically doesn't increment the `commitQueuePosition` by 1 but instead renders a list index and increments that index by 1.

I think I'm kind of being swayed by the fact that both the task resolver and the legacy tasks route use a single function [`FindMinimumQueuePositionForTask`](https://github.com/evergreen-ci/evergreen/blob/main/model/task_queue.go#L539) to get the task queue position for a task. Maybe it could be good if I made a function like `FindCommitQueuePosition` that could be reused by the patch resolver ([here](https://github.com/evergreen-ci/evergreen/blob/main/graphql/patch_resolver.go#L60-L68)) and API ([here](https://github.com/ablack12/evergreen/blob/EVG-17314/rest/model/patch.go#L122-L129), which is Annie's code for [EVG-17314](https://jira.mongodb.org/browse/EVG-17314))? I guess it would only be useful if we want them to match 😥 but I think that could make sense because it might be confusing for users to see two different values... 

sorry for rambling 🧍 I'm okay with doing it on the UI side if you feel strongly about it!